compiler/config.py

```diff
@@ -27,6 +27,8 @@
 
 LOGGING_PREFIX = ""
 
+HDFS_PREFIX = "$HDFS_DATANODE_DIR/"
```

Comment:
Would we ever need to change this? Would it make sense to set this as a `pash` flag?

Reply:
no I don't think we would ever need to change it, so a flag isn't necessary.